src/pages/index.js

```
@@ -1,25 +1,191 @@
+    const filterContent = (topic, content) => {
+        let filtered = content;
+        if (topic !== 'Top' && topic !== 'More') {
+            const topicLower = topic.toLowerCase();
+            if (['courses', 'videos', 'documents'].includes(topicLower)) {
+                filtered = content.filter(item => item.type === topicLower.slice(0, -1));
+            } else if (topicLower === 'free') {
+                filtered = content.filter(item => !item.price || Number(item.price) === 0);
+            } else if (topicLower === 'paid') {
+                filtered = content.filter(item => item.price && Number(item.price) > 0);
+            } else {
+                filtered = content.filter(item => item.topics && item.topics.includes(topicLower));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)